diff -rc g:\tmp\Plotmtv1.4.1/Imakefile plotmtv1.4.1/Imakefile
*** g:\tmp\Plotmtv1.4.1/Imakefile	Sat Nov  1 18:22:30 1997
--- plotmtv1.4.1/Imakefile	Sat May  8 13:08:36 1999
***************
*** 35,41 ****
  	@echo ""
  	@echo "Building Plotmtv Version 1.4.0"
  	@echo ""
! 	@date
  	@echo ""
  	-$(MV) Makefile Makefile.bak
  	$(MAKE) -f Makefile.bak Makefile
--- 35,41 ----
  	@echo ""
  	@echo "Building Plotmtv Version 1.4.0"
  	@echo ""
! 	@unixdate
  	@echo ""
  	-$(MV) Makefile Makefile.bak
  	$(MAKE) -f Makefile.bak Makefile
***************
*** 44,50 ****
  	$(MAKE) $(WORLDOPTS)
  	$(MAKE) install.local
  	@echo ""
! 	@date
  	@echo ""
  	@echo "Full build of Plotmtv Version 1.4.0 complete."
  	@echo ""
--- 44,50 ----
  	$(MAKE) $(WORLDOPTS)
  	$(MAKE) install.local
  	@echo ""
! 	@unixdate
  	@echo ""
  	@echo "Full build of Plotmtv Version 1.4.0 complete."
  	@echo ""
diff -rc g:\tmp\Plotmtv1.4.1/Lib/Imakefile plotmtv1.4.1/Lib/Imakefile
*** g:\tmp\Plotmtv1.4.1/Lib/Imakefile	Fri Jul 14 21:06:38 1995
--- plotmtv1.4.1/Lib/Imakefile	Sat May  8 14:59:50 1999
***************
*** 21,27 ****
    SYS_LIBRARIES = -lm
          DEFINES = 
  
! SOURCES = array.c \
            list1.c \
            list2.c \
            list3.c \
--- 21,28 ----
    SYS_LIBRARIES = -lm
          DEFINES = 
  
! SOURCES = s_erf.c \
!           array.c \
            list1.c \
            list2.c \
            list3.c \
***************
*** 77,83 ****
            system.c \
            time.c 
  
! OBJECTS = array.o \
            list1.o \
            list2.o \
            list3.o \
--- 78,85 ----
            system.c \
            time.c 
  
! OBJECTS = s_erf.o \
!           array.o \
            list1.o \
            list2.o \
            list3.o \
diff -rc g:\tmp\Plotmtv1.4.1/Lib/probplot.c plotmtv1.4.1/Lib/probplot.c
*** g:\tmp\Plotmtv1.4.1/Lib/probplot.c	Fri Jul 14 21:06:52 1995
--- plotmtv1.4.1/Lib/probplot.c	Sat May  8 16:07:50 1999
***************
*** 9,14 ****
--- 9,19 ----
  #include <stdlib.h>
  #include <math.h>
  
+ #ifdef __EMX__
+ #include <float.h>
+ double erf(double);
+ #endif
+ 
  #define ERRORRETURN -9.0e30
  #define MAXITER  3000
  #ifndef EXIT_FAILURE
diff -rc g:\tmp\Plotmtv1.4.1/Lib/readdata.c plotmtv1.4.1/Lib/readdata.c
*** g:\tmp\Plotmtv1.4.1/Lib/readdata.c	Fri Jul 14 21:19:38 1995
--- plotmtv1.4.1/Lib/readdata.c	Sat May  8 13:27:26 1999
***************
*** 198,203 ****
--- 198,226 ----
     /* Initialize */
     *ispipe = CN_FALSE;
  
+ 
+ #ifdef __EMX__
+    if ((strlen(fname) > 4) && (!strcmp(fname + strlen(fname) - 4,".zip"))) {
+      (void) sprintf(cmd,"unzip -p %s",fname);
+      *fp = popen(cmd,"rt");
+      *ispipe = CN_TRUE;
+    } else if (
+ 	      ((strlen(fname) > 2) && (!strcmp(fname + strlen(fname) - 2,".z"))) ||
+ 	      ((strlen(fname) > 3) && (!strcmp(fname + strlen(fname) - 3,".gz"))) ) {
+      (void) sprintf(cmd,"gzip -d -c %s",fname);
+      *fp = popen(cmd,"rt");
+      *ispipe = CN_TRUE;
+    }
+    else if (
+ 	    (strlen(fname) > 2) && (!strcmp(fname + strlen(fname) - 2,".Z"))) {
+      (void) sprintf(cmd,"compress -d -c %s",fname);
+      *fp = popen(cmd,"rt");
+      *ispipe = CN_TRUE;
+    }
+    else {
+      *fp = fopen(fname,"rt");
+    }
+ #else
     /* try to open a file for reading */
     if ((strlen(fname) > 2) && (!strcmp(fname + strlen(fname) - 2,".Z"))) {
        (void) sprintf(cmd,"uncompress -c %s",fname);
***************
*** 212,217 ****
--- 235,241 ----
     } else {
        *fp = fopen(fname,"r");
     }
+ #endif
  
     if (*fp == NULL) {
        (void) fprintf(stderr,"cat : couldn't open file %s\n",fname);
Only in plotmtv1.4.1/Lib: s_erf.c
diff -rc g:\tmp\Plotmtv1.4.1/Plot/test_plotmtv.c plotmtv1.4.1/Plot/test_plotmtv.c
*** g:\tmp\Plotmtv1.4.1/Plot/test_plotmtv.c	Sat Nov  1 18:21:52 1997
--- plotmtv1.4.1/Plot/test_plotmtv.c	Sat May  8 13:30:08 1999
***************
*** 248,255 ****
  		 "\nWelcome to Jai's Special Version of Plotmtv1.4.1");
    (void) fprintf(stdout,
  		 "\nAustin J. Lee 03/23/97\n");
  }
- 
  
  /* 
   * Read the command-line options as well as data.
--- 248,256 ----
  		 "\nWelcome to Jai's Special Version of Plotmtv1.4.1");
    (void) fprintf(stdout,
  		 "\nAustin J. Lee 03/23/97\n");
+   (void) fprintf(stdout,
+ 		 "\nOS/2 patch Th. Hoffmann 08-may-99\n\n");
  }
  
  /* 
   * Read the command-line options as well as data.
Only in plotmtv1.4.1/Utils/TESTS: run.cmd
